libs/designsystem/src/lib/components/action-sheet/action-sheet.component.scss

```diff
@@ -4,20 +4,35 @@
   position: relative;
   display: inline-block;
   outline: none;
+
+  // Only apply focus ring if pointer device can hover
+  // (effectively desktop/mouse devices):
+  @include focus() {
+    > button,
+    > button.attention-level3 {
+      border-color: $focus-ring-color;
+    }
+  }
```

Comment:
Somehow this still applies even when comp is disabled? 🤔 
![image](https://user-images.githubusercontent.com/9556693/119137946-a0e5fe00-ba41-11eb-9260-39395bf29b05.png)